src/lory.js

```diff
@@ -134,7 +134,9 @@ export function lory (slider, opts) {
             rewind,
             rewindSpeed,
             ease,
-            classNameActiveSlide
+            classNameActiveSlide,
+            classNameDisabledNextCtrl = 'disabled',
```

Comment:
Couldn't this be something like `classNameDisabledCtrl` ? (instead of having two variables)

Reply:
It's more flexible, particularly in BEM context: `next-button--disabled` && `prev-button--disabled`.